src/styles/blocks/page.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../../fonts/Avenir.ttc);
+  font-weight: 400;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can specify a list of comma-separated fonts to be used, ending with a generic font family, in case none of the specified fonts are available. For example: `font-family: Avenir, Arial, sans-serif;`.